staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -33,6 +34,15 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// DefaultAuthorizationWebhookRetryBackoff is the default backoff parameters for
+// authorization webhook used by the apiserver.
+var DefaultAuthorizationWebhookRetryBackoff = wait.Backoff{
```

Comment:
Maybe we can collapse both `DefaultAuthorizationWebhookRetryBackoff` and `DefaultAuthenticationWebhookRetryBackoff` into a shared backoff `DefaultWebhookRetryBackoff`.